src/python_bindings/bindings.cpp

```diff
@@ -156,7 +156,57 @@ PYBIND11_MODULE(desbordante, module) {
             .def("get_num_error_rows", &PyFDVerifier::GetNumErrorRows)
             .def("get_highlights", &PyFDVerifier::GetHighlights);
 
-    DEFINE_ALGORITHM(DataStats, Algorithm).def("get_result_string", &PyDataStats::GetResultString);
+    DEFINE_ALGORITHM(DataStats, Algorithm)
+            .def("get_result_string", &PyDataStats::GetResultString)
+            .def("get_number_of_values", &PyDataStats::GetNumberOfValues,
+                 "Get number of values in column.")
+            .def("get_number_of_columns", &PyDataStats::GetNumberOfColumns,
+                 "Get number of columns in table.")
+            .def("get_columns_with_null", &PyDataStats::GetColumnsWithNull,
+                 "Get number columns with null.")
+            .def("get_columns_with_unique_values", &PyDataStats::GetColumnsWithUniqueValues,
+                 "Get number of columns with unique values.")
+            .def("get_number_of_distinct", &PyDataStats::Distinct,
+                 "Get number of unique values in the column.")
+            .def("is_categorical", &PyDataStats::IsCategorical,
+                 "Check if quantity <= count of unique values in the column.")
+            .def("show_sample", &PyDataStats::ShowSample, py::arg("start_row"), py::arg("end_row"),
+                 py::arg("start_col"), py::arg("end_col"), py::arg("str_len") = 10,
+                 py::arg("usigned_len") = 5, py::arg("double_len") = 10,
+                 "Returns table slice from start_row to end_row and from start_col to end_col. "
+                 "Data values converted to string type.")
```

Comment:
Are you sure this method is supposed to be exposed to the user? That length specification makes me doubt it.

Reply:
It's nothing else but giving a sample to user and length specification just tells how many signs will be used to display value in the cell for each value type. Like if there are long string in table we may shorten them with `str_len = 5`